modules/templateRenderer.js

```diff
@@ -176,9 +176,11 @@
 
     hotswapKeepCaret: function(currentNode, newNode, ignoreElements) {
       var currentCaret,
-          activeElement = document.activeElement;
-      if (activeElement && this.supportsSelection(activeElement)) {
-        currentCaret = this.getCaretPosition(activeElement);
+          activeElement;
+      try {
+        activeElement = document.activeElement;
```

Comment:
Gah - deleted more lines than I meant to!  Thank you for catching; I will fix immediately.